dev/dependencies.sh

```diff
@@ -56,9 +56,9 @@ function mvn_build_classpath() {
 }
 
 function sbt_build_client_classpath() {
-  PATTERN="$SBT_PROJECT / Runtime / externalDependencyClasspath"
+  PATTERN="$SBT_PROJECT / Runtime / managedClasspath"
```

Comment:
cc @cfmcgrady

Reply:
From SBT docs

[externalDependencyClasspath](https://github.com/sbt/sbt/blob/edc10297b0977e3b099f37b8363b33b404bc43c3/main/src/main/scala/sbt/Keys.scala#L366)

> The classpath consisting of library dependencies, both managed and unmanaged.

[managedClasspath](https://github.com/sbt/sbt/blob/edc10297b0977e3b099f37b8363b33b404bc43c3/main/src/main/scala/sbt/Keys.scala#L364)

> The classpath consisting of external, managed library dependencies.

Can you explain more about why we need this change?

Reply:
@cfmcgrady, I have introduced `JDKTools` to add `tool.jar` into `unmanagedJars`, which causes that the path of `tool.jar` is relative. Meanwhile, the maven dependencies are all added into  `managedJars`, therefore we could only check the managed classpath.

Reply:
Thank you for your detailed explanation.